infra/gp-external-secrets-operator/templates/01_namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    kubernetes.io/metadata.name: external-secrets-operator
+    openshift.io/cluster-monitoring: "true"
+  name: gp-eso
```

Comment:
Ich würde den Name Konfigurierbar machen, wenn du ihn mal ändern willst musst du das in allen Files händisch machen. Dann an allen weitern Stellen wo namespace: gp-eso vorkommt nachziehen.
Andere Frage: Muss der Namespace extra angelegt werden, oder reicht es nicht einfach überall {{ .Release.Namespace }} zu verwenden und in Argocd CreateNamespace auf true setzen?

Reply:
Hast recht, hab ich nach meinen lokalen Installationen nicht rausgenommen, aber ist mit Argo Deployment redundant